contracts/truefi2/Liquidator2.sol

```diff
@@ -181,4 +185,17 @@ contract Liquidator2 is UpgradeableClaimable {
     function getDefaultedValueInUsd(uint256 defaultedValue, ITrueFiPoolOracle oracle) internal view returns (uint256) {
         return oracle.tokenToUsd(defaultedValue);
     }
+
+    function areLoansOfSingleBorrower(IDebtToken[] calldata loans) internal view returns (bool) {
+        if (loans.length == 1) {
```

Comment:
If we pass in an empty list this is vacuously true.
```suggestion
        if (loans.length <= 1) {
```
I think we don't need to revert on empty lists (an empty list shouldn't hurt, it just wastes the caller's gas). But maybe we should add a unit test to confirm an empty list doesn't transfer any TRU.